### ISSUE TITLE
13.11. Goal 2 - Get a model to train

### ISSUE THREAD

**Issue by phinik** [phinik/LovelyLLamas#8]
Set up a first training pipeline

To implement:
1. Dataset class
2. Dataloaders
3. DummyTokenizer
4. Trainer class

To be used:
1. Transformer model
6. CE loss

**Comment by phinik** [comment]
Transformer does train, but not really converge for know. Maybe just too little training data?

**Comment by phinik** [comment]
It now converges and overfits the little training data after about 5 epochs
